Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gcr.io/cloud-builders/gcloud
```

Comment:
Outside the context of this PR, it is not clear what this Dockerfile is for, consider adding comments. Also, why this file is not under cloud-build?

Reply:
The docker build uses the Dockerfile found at the root of the repo. I tried to place it in cloud-build and use that moved Dockerfile instead by setting the 'dir' attribute of the step, but that made it impossible to access/copy the init actions from the parent directory.

Reply:
I'm looking more into this.

Reply:
The Dockerfile is inside now.